pdp/static/js/pdp_controls.js

```diff
@@ -179,6 +180,35 @@ Colorbar.prototype = {
         }
     },
 
+    round_to_nearest: function (mult) {
+      this.minimum = Math.round(this.minimum / mult) * mult;
+      this.maximum = Math.round(this.maximum / mult) * mult;
+      this.midpoint = Math.round(this.midpoint / mult) * mult;
+    },
+
+    apply_precision: function (sig) {
+      this.minimum = this.minimum.toFixed(sig);
+      this.maximum = this.maximum.toFixed(sig);
+      this.midpoint = this.midpoint.toFixed(sig);
+    },
+
+    estimate_precision: function () {
+      var range = this.maximum - this.minimum;
```

Comment:
Could you add some comments here to describe the intent of the return value? You're kind of doing two things here: mutating state and returning a value. It would be better to perform one (and only one) operation. Also, why not pass in min/max as parameters, instead of retrieving from properties? Try and use as little state as possible. In this case you don't need any.